data/announcements/example.yaml

```diff
@@ -0,0 +1,28 @@
+---
+### Example announcement
+#
+# Leave this example announcement in place. It serves as a template
```

Comment:
@sftim, 
Is the plan to add all announcement data to a single file or use one file per announcement?
If the data is placed in a single file then could add extra data to specify which announcements to publish (no priority variable). How many announcements can be published simultaneously?

```
---
publish:
   - name: "kubecon europe"
   - name: "testing"
announcements:
   - announcement:
      name: "test"
      startTime: 2020-01-01T00:00:00
      endTime: 2020-04-01T00:00:00
      title: Sample 1 announcement
      message: |-
       Message 1.
   - announcement:
      name: "blm"
      startTime: 2020-01-01T00:00:00
      endTime: 2020-04-01T00:00:00
      style: >-
       color: #fff; background: #000;
      title: Sample 2 announcement
      message: |-
       Message 2.
   - announcement:
      name: "kubecon europe"
      startTime: # this defaults to now
      endTime: 2020-04-01T00:00:00
      title: Sample 3 announcement
      message: |-
       Message 3.
       Messages must contain less than 3 lines of text.
###
```